src/frontend/sharedTypes/navigation.ts

```diff
@@ -43,7 +43,16 @@ export type RootStackParamsList = {
   CoordinateFormat: undefined;
   Experiments: undefined;
   PhotoPreviewModal: {
+    createdByDeviceId?: string;
+    observationDocId?: string;
     photo: SavedPhoto | ProcessedDraftPhoto;
+    validatedByCoMapeo?: boolean;
+  };
+  ConfirmDeletePhoto: {
+    onSuccess?: () => void;
```

Comment:
doing this aligns with our agreed pattern of allowing callback nav params for certain kinds of screens.

In a separate branch, I experimented with a non-callback based approach that seems to work, and aligns with some of the suggestions that React Navigation.

https://github.com/digidem/comapeo-mobile/commit/a4eb915d6914716108caa43cb7aa620c40fae6ca

Not sure if it's really that much better, but it works and isn't overly complicated.